kubernetes/cray-dhcp-kea/values.yaml

```diff
@@ -188,9 +242,26 @@ crayDhcpKeaCtrlAgentConfig:
         output_options:
           - output: stdout
         severity: ERROR
-global:
-  chart:
-    name: cray-dhcp-kea
-    version: 0.10.25
-  appVersion: 0.10.25
 
+cray-postgresql:
```

Comment:
If you want the db backed up, you will need to add that here.

Something like this:
    enableLogicalBackup: true
    logicalBackupSchedule: "10 0 * * *"  # Once per day at 12:10AM

Reply:
We don't want to back it up